front-end/src/components/Login/Web3Login.tsx

```diff
@@ -155,26 +160,30 @@ const LoginForm = ({ className, toggleWeb2Login }:Props): JSX.Element => {
 				</div>
 				: null
 			}
-			{accounts.length > 0 &&
-				<Form.Group>
-					<AccountSelectionForm
-						title='Choose linked account'
-						accounts={accounts}
-						address={address}
-						onAccountChange={onAccountChange}
-					/>
-				</Form.Group>
-			}
-			{!extensionNotFound &&
-				<div className={'mainButtonContainer'}>
-					<Button
-						primary
-						disabled={loading}
-						type='submit'
-					>
-						Login
-					</Button>
-				</div>
+			{isAccountLoading
```

Comment:
The loader is above the login switch button unfortunately. You can set a `height: 10rem` and it should be better.
That's how it is now.
![image](https://user-images.githubusercontent.com/33178835/78646788-dff6d580-78b9-11ea-86a8-a1c1d0db3a27.png)